site/source/pages/examples/spec-inlined.hbs

```diff
@@ -5,224 +5,199 @@ layout: example.hbs
 
 <h1>Inlined Chart Specification</h1>
 
-<div id="chart"></div>
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+  <script src="//code.jquery.com/jquery-1.10.2.js"></script>
+  <script src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+  <style>
+    #examples { height: 1800px; }
+    #resizable { width: 550px; height: 150px; padding: 0.5em; }
+    #resizable h3 { text-align: center; margin: 0; }
+  </style>
 
+  <div id="resizable" class="ui-widget-content">
+    <div id="chart" style="height: 100%"></div>
+    </div>
 
-{{#markdown}}
-```html
+
+<script>
   //create a cedar chart
   var chart = new Cedar();
 
   //create the specification
   var spec ={
-  "inputs": [
-    {"name": "y", "type": ["numeric","string"], "required": true},
-    {"name": "label", "type": ["string"], "required": false},
-    {"name": "radius", "type": ["numeric"], "required": true}
+  "inputs":[
+    {"name":"x","type":["time"], "required":true},
+    {"name":"y","type":["numeric"], "required":true}
   ],
   "template":{
     "padding": "strict",
     "data": [
       {
         "name": "table",
-        "format": {
-          "property": "features"
-        },
+        "url":"{data}&orderByFields={x.field}",
+        "format": {"type": "json", "property": "features", "parse": {"attributes.{x.field}":"date"}}
+      },
+      {
+        "name": "exports",
+        "source": "table",
         "transform": [
-          {"type": "pie", "value": "attributes.{y.field}"},
-          {"type": "formula", "field": "hyp", "expr": "{radius}"},
-          {"type": "formula", "field": "theta", "expr": "d.startAngle + 0.5 *(d.endAngle - d.startAngle) - 1.57079633"},
-          {"type": "formula", "field": "x_mid", "expr": "{radius} / 2"},
-          {"type": "formula", "field": "y_mid", "expr": "{radius} / 2"},
-          {"type": "formula", "field": "x", "expr": "d.x_mid + (d.hyp * Math.cos(d.theta))"},
-          {"type": "formula", "field": "y", "expr": "d.y_mid + (d.hyp * Math.sin(d.theta))"}          
+          {"type": "formula", "field": "export", "expr": "datum"}
         ]
       }
     ],
     "scales": [
       {
-        "name": "r",
-        "type": "sqrt",
+        "name": "x",
+        "type": "time",
+        "range": "width",
+        "nice": "month",
```

Comment:
It is probably OK here because it's just a sample. But why is it set to `month` in the default template:
https://github.com/Esri/cedar/blob/develop/src/charts/time.json#L27

Or does it automatically switch to `year` when data spans several years and there is not enough room to render labels month by month?